websites/S/Scantrad/presence.ts

```diff
@@ -0,0 +1,55 @@
+const presence = new Presence({
+    clientId: "817856303719907389"
+}),
+    browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = { 
+        largeImageKey: "logo",
+        startTimestamp: browsingStamp
+    };
+    if (document.location.pathname.includes("/")) {
+        presenceData.details = "Viewing Page:";
+        presenceData.state = "Home page";
+        presenceData.startTimestamp = browsingStamp;
+        };
+    if (document.location.pathname.includes("/mangas")) {
+
+        let chapter = document.getElementById("selectCh") as HTMLSelectElement,
+            page = document.getElementById("selectPg") as HTMLSelectElement;
```

Comment:
What's the point of letting this when it's never reassigned?

Reply:
It's actually reassigned?

Reply:
it is not

Reply:
codacy agrees with me